.github/workflows/docker-develop.yml

```diff
@@ -0,0 +1,45 @@
+name: siglens-docker-develop-build
```

Comment:
rename this file to be `develop-e2e-install.yml`
accordingly rename the `name` field in here to be `develop-e2e-install-test`